### ISSUE TITLE
Incompatible with Kotlin 1.5.20-M1

### ISSUE THREAD

**Issue by ZacSweers** [google/ksp#481]
If I try to use beta02 with Kotlin 1.5.20-M1, this happens

```
Caused by: java.lang.NoClassDefFoundError: org/jetbrains/kotlin/gradle/tasks/KotlinCompileTaskData
        at com.google.devtools.ksp.gradle.InternalTrampoline.KotlinCompileTaskData_register(InternalTrampoline.java:12)
        at com.google.devtools.ksp.gradle.KspGradleSubplugin.applyToCompilation(KspSubplugin.kt:224)
        at org.jetbrains.kotlin.gradle.plugin.SubpluginEnvironment.addSubpluginOptions(SubpluginEnvironment.kt:80)

```

**Comment by neetopia** [comment]
There is breaking change in upstream for 1.5.20, #480 might contain a fix to this.

**Comment by ting-yuan** [comment]
Yep. Fixed in #480. It will be available in the next release, which targets at 1.5.20.
